Testscripts/Linux/utils.sh

```diff
@@ -2428,6 +2428,13 @@ function install_iperf3 () {
 	ip_version=$1
 	LogMsg "Detected $DISTRO_NAME $DISTRO_VERSION; installing required packages of iperf3"
 	update_repos
+
+#if 'ip_version" is not passed assume its ipv4
+	if [[ "x$ip_version" =~ "x" ]]
+	then
```

Comment:
remove ~?

Reply:
Its tested and working fine.

Reply:
![image](https://user-images.githubusercontent.com/10083705/119318863-5fa55680-bcac-11eb-84d9-f469fd475bf3.png)

Reply:
@SRIKKANTH Could you please test in above 3 situations? when ip_version is empty, 4, 6, all will enter into if, that is unexpected.